main.js

```diff
@@ -0,0 +1,95 @@
+const MenuSection = document.getElementById('Menu');
+
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Well, I've noticed that you have disabled some ESLint linter errors. It's important to note that this is generally not considered a good practice. I strongly recommend that you remove all ESLint disable comments and instead go through the linter output, addressing each error individually. ✅🔍 Linters play a crucial role in identifying potential issues and enhancing code consistency. By addressing these linter errors, you'll improve the readability and maintainability of your code. Keep up the great work! 🚀💻